spec/factories/order_detail.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :order_detail do
+    current_price {Faker::Number.decimal(l_digits: 2)}
+    quantity {rand(1..10)}
```

Comment:
thêm association vào thôi em nhé ;)
```suggestion
FactoryBot.define do
  factory :order_detail do
    current_price {Faker::Number.decimal(l_digits: 2)}
    quantity {rand(1..10)}
    order
  end
```

Reply:
Dạ. Em hiểu rồi. Cám ơn chị Thảo.